yt_dlp/extractor/radio4dk.py

```diff
@@ -0,0 +1,38 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    unescapeHTML,
+    unified_strdate,
+)
+
+
+class Radio4DkIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?radio4\.dk/program/[^/]+/\?.*\bgid=(?P<id>\d+)\b'
+
+    _TEST = {
```

Comment:
`_TESTS = [{...}]`

Reply:
Okay, obviously "legacy", because I copy/pasted from a different extractor. ;) (Yes, I did notice that CONTRIBUTING.md is clearer about this.)